design-system/uilib/src/components/dpdhl-radio-group/dpdhl-radio-group.svelte

```diff
@@ -0,0 +1,58 @@
+<script context="module" lang="ts">
+	export const ITEMS = {};
+</script>
+
+<script lang="ts">
+	import { setContext, onDestroy } from 'svelte';
+	import { writable } from 'svelte/store';
+	import { createEventDispatcher } from 'svelte';
+	import { get_current_component } from "svelte/internal";
+
+	const items = [];
+	let selectedItem = writable(null);
+
+	const component = get_current_component()
+	const svelteDispatch = createEventDispatcher()
+
+	export function dispatch(name, detail = null) {
+		svelteDispatch(name, detail)
+		component.dispatchEvent && component.dispatchEvent(new CustomEvent(name, { detail }))
```

Comment:
why do we dispatch a svelte event and a DOM event?
would be the DOM event enough? Don't we loose the svelte event when we compile to web-components?